### DISCUSSION TITLE
Breadcrumb - how to select last:child for styling

### DISCUSSION THREAD

**Discussion by jennifergithub** [microsoft/fluentui#31981] [General] [answered]
Hi, I can't seem to style the last itemLink. 
<img width="750" alt="image" src="https://github.com/microsoft/fluentui/assets/62663431/e477f6cb-dba2-454c-8a4b-322cf950b5fe">

You can see I've successfully selected for the last child, but I'm unable to override the default CSS settings. How do I select for the last itemLink for styling? Thanks.

![image](https://github.com/microsoft/fluentui/assets/62663431/bc4539fa-bf0d-4f7b-8764-e0130e76e655)

**Comment by spmonahan** [comment, marked as answer]
Because the existing styles references the `.ms-Breadcrumb-itemLink` classname I think you'll need to use it as well:

```js
const breadcrumbStyles = {
  listItem: {
    ':last-child .ms-Breadcrumb-itemLink': {
        fontWeight: 100
    }
  }
}```
